src/Appwrite/Utopia/Response.php

```diff
@@ -472,7 +486,7 @@ public function yaml(array $data): void
     /**
      * @return array
      */
-    public function getPayload(): array
+    public function getPayload(): string | array
```

Comment:
This is usually a bad practice, why do we need it to return a string?

Reply:
The first ver was a string then i changed to [].
Fixed it!

Reply:
changed

Reply:
I can still see both types.